src/main/resources/init_db.sql

```diff
@@ -0,0 +1,9 @@
+CREATE DATABASE IF NOT EXISTS test;
+
+USE test;
+
```

Comment:
how about schema?

I believe you should create schema if not exists, not a database

correct me if I'm wrong